src/types/api/tenant.ts

```diff
@@ -45,7 +45,6 @@ export interface TTenant {
     StorageAllocatedSize?: string; // Actual database size
     /** uint64 */
     StorageMinAvailableSize?: string;
-    Nodes?: TSystemStateInfo[];
```

Comment:
`Nodes` and `MonitoringEndpoint` requires additional param in endpoint (`nodes=true`), we don't sent this param and thus never have these fields in response